core/src/main/java/com/orientechnologies/orient/core/db/document/ODatabaseDocumentEmbedded.java

```diff
@@ -661,27 +683,40 @@ public OResultSet execute(String language, String script, Map<String, ?> args) {
       checkSecurity(ORule.ResourceGeneric.COMMAND, ORole.PERMISSION_EXECUTE, language);
     }
 
-    OScriptExecutor executor =
-        sharedContext
-            .getOrientDB()
-            .getScriptManager()
-            .getCommandManager()
-            .getScriptExecutor(language);
-    OResultSet original;
-
-    ((OAbstractPaginatedStorage) this.storage.getUnderlying())
-        .pauseConfigurationUpdateNotifications();
+    if (OGlobalConfiguration.COMMAND_TIMEOUT.getValueAsInteger() > 0) {
```

Comment:
instead of: `OGlobalConfiguration.COMMAND_TIMEOUT.getValueAsInteger() ` we should use ` getContextConfiguraion().getValueAsInteger(OGlobalConfiguration.COMMAND_TIMEOUT) `

Reply:
In the database actually the configuration is in the `getConfiguration` not in `getContextConfiguration` so it should be: `getConfiguration().getValueAsInteger(OGlobalConfiguration.COMMAND_TIMEOUT)`

Reply:
Done